pkg/testutils/lint/lint_test.go

```diff
@@ -2794,4 +2802,51 @@ func TestLint(t *testing.T) {
 			}
 		}
 	})
+
+	// Test forbidden roachtest imports.
+	t.Run("TestRoachtestForbiddenImports", func(t *testing.T) {
```

Comment:
It would be nice to add a unit test. You can easily adapt one from `TestForbiddenImports`.

Reply:
> Am I missing something obvious here, or a way to test the linter unit test?

Never mind, I thought we already had a unit test for `TestForbiddenImports`, but we only have unit tests for standalone linters, e.g., `unconvert_test.go`.